### PR TITLE
docs: Moved Trace Viewer running instructions to the top of the page

### PR DESCRIPTION
**Pull request by agg23**
As discussed internally, moved instructions to open Trace Viewer and create traces to the top of the docs page, matching UI Mode's docs. Additionally cleaned up some grammar on UI Mode introduction.

### CODE REVIEW

File: docs/src/trace-viewer.md [hunk 1]
```diff
@@ -17,106 +17,67 @@ Playwright Trace Viewer is a GUI tool that helps you explore recorded Playwright
...
-Next to the Actions tab you will find the Metadata tab which will show you more information on your test such as the Browser, viewport size, test duration and more.
+When using [trace.playwright.dev](https://trace.playwright.dev), you can also pass the URL of your uploaded trace at some accessible storage (e.g. inside your CI) as a query parameter. CORS (Cross-Origin Resource Sharing) rules might apply.
 
-![meta data in trace viewer](https://github.com/microsoft/playwright/assets/13063165/82ab3d33-1ec9-4b8a-9cf2-30a6e2d59091)
+```txt
+https://trace.playwright.dev/?trace=https://demo.playwright.dev/reports/todomvc/data/cb0fa77ebd9487a5c899f3ae65a7ffdbac681182.zip
+```
 
-### Attachments
+## Recording a trace
```
Comment (yury-s) [inline docs/src/trace-viewer.md:74]: Should this section go before `Opening Trace Viewer` similar to `trace-viewer-intro.md`?

File: docs/src/trace-viewer.md [hunk 2]
```diff
@@ -17,106 +17,67 @@ Playwright Trace Viewer is a GUI tool that helps you explore recorded Playwright
...
+playwright show-trace trace.zip
+```
 
-See console logs from the browser as well as from your test. Different icons are displayed to show you if the console log came from the browser or from the test file.
+```bash csharp
+pwsh bin/Debug/netX/playwright.ps1 show-trace trace.zip
+```
 
-![showing log of tests in trace viewer](https://github.com/microsoft/playwright/assets/13063165/4107c08d-1eaf-421c-bdd4-9dd2aa641d4a)
+## Using [trace.playwright.dev](https://trace.playwright.dev)
```
Comment (yury-s) [inline docs/src/trace-viewer.md:40]: Maybe make it a subsection?

File: docs/src/test-ui-mode-js.md
Comment (yury-s) [inline docs/src/test-ui-mode-js.md:1]: nice!

**Review by yury-s — APPROVED** [review]

File: docs/src/trace-viewer.md (hunk 1 above)
Comment (agg23) [reply docs/src/trace-viewer.md:74]: I went back and forth on this. My opinion is that `trace-viewer-intro.md` is a how-to guide, and `trace-viewer.md` are the actual docs, so they don't really need to correlate with each other. Matching UI Mode docs style, I think the instructions on how to open it should come first, as that's what people are most likely to look up at any moment (you set up traces once, you open traces many times).

File: docs/src/trace-viewer.md (hunk 2 above)
Comment (agg23) [reply docs/src/trace-viewer.md]: Good point. Done.

**Pull request merged**